cmd/cortex/main.go

```diff
@@ -27,14 +27,20 @@ func init() {
 
 func main() {
 	var (
-		cfg             cortex.Config
-		configFile      = ""
-		eventSampleRate int
-		ballastBytes    int
+		cfg                  cortex.Config
+		configFile           = ""
+		eventSampleRate      int
+		ballastBytes         int
+		mutexProfileFraction int
 	)
 	flag.StringVar(&configFile, "config.file", "", "Configuration file to load.")
 	flag.IntVar(&eventSampleRate, "event.sample-rate", 0, "How often to sample observability events (0 = never).")
 	flag.IntVar(&ballastBytes, "mem-ballast-size-bytes", 0, "Size of memory ballast to allocate.")
+	flag.IntVar(&mutexProfileFraction, "mutex-profile-fraction", 0, "Fraction at which mutex profile vents will be reported, 0 to disable")
```

Comment:
What if we prefix it with `debug.` (so the flag will be `-debug.mutex-profile-fraction`) to explicitly state it's a flag just used for debugging?